subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -6196,6 +6197,11 @@ int lwm2m_engine_send(struct lwm2m_ctx *ctx, char const *path_list[], uint8_t pa
 	sys_slist_t lwm2m_path_list;
 	sys_slist_t lwm2m_path_free_list;
 
+	if (lwm2m_server_get_mute_send(ctx->srv_obj_inst)) {
+		LOG_WRN("Send operation is muted by server");
+		return -EOPNOTSUPP;
```

Comment:
Up for discussion what errno code would be the best fit here, but maybe `EPERM` would be a better choice (not permitted)?

Reply:
I was also thinking same. I can change to EPERM